static/app/components/replays/breadcrumbs/openReplayComparisonButton.tsx

```diff
@@ -39,7 +43,12 @@ export function OpenReplayComparisonButton({
               fallback={
                 <Fragment>
                   <deps.Header closeButton>
-                    <deps.Header>{t('Hydration Error')}</deps.Header>
+                    <deps.Header>
+                      <h4>
+                        Hydration Error
```

Comment:
does this need to be translated?

Reply:
I un-translated it because on the server we're creating errors with the english name "Hydration Error".
https://github.com/getsentry/sentry/blob/9f3bfebb247fe6ac67a67831f7d3138545babf64/src/sentry/replays/usecases/ingest/issue_creation.py#L13-L19

Also, help text for hydration errors are not translated:
https://github.com/getsentry/sentry/blob/9f3bfebb247fe6ac67a67831f7d3138545babf64/static/app/utils/issueTypeConfig/errorConfig.tsx#L201-L215

And the modal loading state doesn't have it translated:
https://github.com/getsentry/sentry/blob/9f3bfebb247fe6ac67a67831f7d3138545babf64/static/app/components/replays/breadcrumbs/replayComparisonModal.tsx#L32-L35